common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/VroPackageStore.java

```diff
@@ -375,4 +393,31 @@ private String createBackupFilePath(final Package pkg, final String currentDateT
 
 		return newFullPath;
 	}
+
+	/**
+	 * Compares packages by their version and return the highest version package.
+	 * @param packages the collection of packages differring by version
+	 * @return the package with highest version
+	 */
+	private Package getHighestVersion(List<Package> packages) {
+
+		Package highestVersionPac = packages.get(0);
+		if (packages.size() == 1) { 
+			System.out.println("Highest version package to backup: " + highestVersionPac.getFQName());
+			return highestVersionPac;
+		}
+		
+		for (int i = 0; i < packages.size() - 1; i++) {
+
+			int diff = highestVersionPac.compareTo(packages.get(i + 1));
+			if (diff > 0) {
+				highestVersionPac = packages.get(i);
+			} else {
+				highestVersionPac = packages.get(i + 1);
+			}
+		}
+		
+		System.out.println("Highest version package to backup: " + highestVersionPac.getFQName());
```

Comment:
Can you please user the logger object to send information instead of system.out?